repl/src/main/scala/ammonite/repl/interp/Compiler.scala

```diff
@@ -117,8 +117,45 @@ object Compiler{
             dirDeps: Seq[java.io.File],
             dynamicClasspath: VirtualDirectory,
             evalClassloader: => ClassLoader,
+            pluginClassloader: => ClassLoader,
             shutdownPressy: () => Unit): Compiler = new Compiler{
 
+    val PluginXML = "scalac-plugin.xml"
+
+    lazy val plugins0 = {
+      import scala.collection.JavaConverters._
+      val loader = pluginClassloader
+
+      val plugins = loader
+        .getResources(PluginXML)
+        .asScala
+        .toVector
+        .map { url =>
+          val elem = scala.xml.XML.load(url.openStream())
+          val name = (elem \\ "plugin" \ "name").text
+          val className = (elem \\ "plugin" \ "classname").text
+          name -> className
+        }
+        .filter{
+          case ("acyclic", _) => false // conflicts with AmmonitePlugin
```

Comment:
I can't make sense of this comment, even though I think I know what it does (avoid loading Acyclic, which lives on the parent classpath).

On the other hand, I don't really see why loading Acyclic as a compiler plugin is a problem? Why does it conflict with AmmonitePlugin?

Is this filter still necessary? If so, we should merge it with the `collect` phase below and add a better description. If not we should kill it